deployability/modules/workflow_engine/tests/TESTING-README.md

```diff
@@ -0,0 +1,159 @@
+# Workflow engine Unit Testing using Pytest
+
+The workflow_engine module includes pytest unit tests.
+
+## Requirements
+
+- Make sure you have Python installed on your system. You can download it from 
+    [python.org](https://www.python.org/downloads/).
+- Clone the wazuh-qa repository in your local environment.
+- Install the necessary dependencies by running:
+```bash
+git clone git@github.com:wazuh/wazuh-qa.git -b [your-branch]
```

Comment:
This is a public repository, maybe we should use `https://github.com/wazuh/wazuh-qa.git` instead of `git@`